src/vm/gcenv.ee.cpp

```diff
@@ -966,6 +975,14 @@ void GCToEEInterface::StompWriteBarrier(WriteBarrierParameters* args)
     default:
         assert(!"unknown WriteBarrierOp enum");
     }
+    if (flushXrestart & ICACHE_FLUSH) 
+    {
+        ::FlushInstructionCache();
```

Comment:
Since address and size calculations are platform-specific, we have to call-back to jitinterfaceXXX code.